package.json

```diff
@@ -21,37 +21,38 @@
   ],
   "scripts": {
     "build": "tsc",
+    "build:map": "tsc --sourceMap",
     "postbuild": "npm run lessc",
     "test": "karma start",
-    "pretest": "npm run build",
+    "pretest": "npm run build:map",
     "lint": "tslint \"+(src|test)/**/*.ts\"",
-    "typings": "typings",
-    "typings:install": "typings install",
     "lessc": "lessc src/styles/style.less lib/index.css"
   },
   "devDependencies": {
+    "@types/d3": "3.5.36",
     "@types/jasmine": "2.5.37",
-    "jasmine": "2.5.2",
+    "@types/jquery": "2.0.41",
+    "@types/lodash": "4.14.55",
+    "coveralls": "2.12.0",
+    "jasmine": "2.5.3",
     "jasmine-jquery": "2.1.1",
-    "karma": "1.3.0",
+    "jquery": "3.2.1",
+    "karma": "1.5.0",
     "karma-chrome-launcher": "2.0.0",
-    "karma-jasmine": "1.0.2",
-    "karma-typescript-preprocessor": "0.3.0",
-    "less": "2.7.1",
+    "karma-coverage": "1.1.1",
+    "karma-jasmine": "1.1.0",
+    "karma-remap-istanbul": "0.6.0",
+    "karma-sourcemap-loader": "0.3.7",
+    "karma-typescript-preprocessor": "0.3.1",
+    "less": "2.7.2",
     "powerbi-visuals-tools": "1.2.1",
     "powerbi-visuals-utils-colorutils": "^0.2.0",
-    "powerbi-visuals-utils-dataviewutils": "^0.3.0",
+    "powerbi-visuals-utils-dataviewutils": "^1.0.0",
     "powerbi-visuals-utils-testutils": "^0.2.0",
-    "tslint": "3.15.1",
-    "typescript": "1.8.10",
-    "typings": "2.0.0",
-    "coveralls": "2.11.15",
-    "karma-coverage": "1.1.1",
-    "karma-remap-istanbul": "0.4.0",
-    "karma-sourcemap-loader": "0.3.7"
+    "tslint": "4.5.1",
```

Comment:
please also add "tslint-microsoft-contrib" and have it execute as part of the 'lint' command.